import pytest


def test_foo():
    pytest.fail("This test is supposed to fail")
